src/components/tabs/tabs.test.js

```diff
@@ -0,0 +1,603 @@
+              ...currentState,
+              error: !currentState.error,
+            }))
+          }
+          checked={error}
+        />
+        <Checkbox
+          label="Add warning"
+          warning={warning}
+          onChange={() =>
+            setValidation((currentState) => ({
+              ...currentState,
+              warning: !currentState.warning,
+            }))
+          }
+          checked={warning}
+        />
+        <Checkbox
+          label="Add info"
+          info={info}
+          onChange={() =>
+            setValidation((currentState) => ({
+              ...currentState,
+              info: !currentState.info,
+            }))
+          }
+          checked={info}
+        />
+      </Tab>
+    </Tabs>
+  );
+};
+
+context("Testing Tabs component", () => {
+  describe("should render Tabs component", () => {
+    it.each([
+      [1, "Content for tab "],
+      [2, "Content for tab "],
+      [3, "Content for tab "],
+      [4, "Content for tab "],
+      [5, "Content for tab "],
+    ])(
+      "should verify when tab %s is clicked that tab is visible and content text is displayed",
+      (id, content) => {
+        CypressMountWithProviders(<TabsComponent />);
+
+        tabById(id).click();
+        tabContentById(id)
+          .should("be.visible")
+          .and("have.text", `${content}${id}`);
+      }
+    );
+
+    it("should verify first tab has a link property", () => {
+      CypressMountWithProviders(
+        <TabsComponent href="https://carbon.sage.com/" />
+      );
+
+      tabById(1)
+        .should("have.attr", "href", "https://carbon.sage.com/")
+        .and("have.attr", "target", "_blank");
+    });
+
+    it("should verify Tabs hidden content is not rendered when renderHiddenTabs prop is false", () => {
+      CypressMountWithProviders(<TabsComponent renderHiddenTabs={false} />);
+
+      tabContentById(1).should("be.visible");
+      tabContentById(2).should("not.exist");
+    });
+
+    it.each([
+      ["tab-5", "Content for tab ", 5],
+      ["tab-4", "Content for tab ", 4],
+      ["tab-3", "Content for tab ", 3],
+      ["tab-2", "Content for tab ", 2],
+      ["tab-1", "Content for tab ", 1],
+    ])(
+      "should verify correct tab is visible and correct content displayed when using selected with selectedTabId prop",
+      (selected, content, id) => {
+        CypressMountWithProviders(<TabsComponent selectedTabId={selected} />);
+
+        tabContentById(id)
+          .should("be.visible")
+          .and("have.text", `${content}${id}`);
+      }
+    );
+
+    it.each([
+      ["right", "have.css"],
+      ["left", "not.have.css"],
+    ])("should verify Tabs component is %s aligned", (alignment, value) => {
+      CypressMountWithProviders(<TabsComponent align={alignment} />);
+
+      tabById(1)
+        .parent()
+        .should(value, "text-align", alignment)
+        .and(value, "justify-content", "flex-end");
```

Comment:
what about `flex-start`?

Reply:
justify-content is 'normal' instead of 'flex-start' when prop is set to 'left' - I've added a line to check for that.